okhttp/src/jvmMain/kotlin/okhttp3/internal/http2/Http2Stream.kt

```diff
@@ -415,8 +416,6 @@ class Http2Stream internal constructor(
         }
 
         if (readBytesDelivered != -1L) {
-          // Update connection.unacknowledgedBytesRead outside the synchronized block.
-          updateConnectionFlowControl(readBytesDelivered)
```

Comment:
Ooooh changes like this make me want to double- or triple- check our tests

Reply:
That's next.